src/menubar.ts

```diff
@@ -187,6 +201,51 @@ export default class UI extends EventEmitter {
     });
   }
 
+  private notifyForIncidentsPastTwoDays(): void {
+    const recentIncidents = this.incidentFeed.getFeed().filter(item => {
+      const publicationDate = new Date(item.pubDate);
+      return isToday(publicationDate) || isYesterday(publicationDate);
+    });
+    if (recentIncidents.length) {
```

Comment:
These three if conditions do more or less the same. Would a method like 

```
notifyAboutIncident({
  incident,
  title: 'New incident reported'
})
```

make sense?